frontend/src/app/admin/Organization/PendingOrganizationsList.tsx

```diff
@@ -0,0 +1,187 @@
+import { gql, useMutation } from "@apollo/client";
+import React, { useState } from "react";
+import classnames from "classnames";
+import { toast } from "react-toastify";
+
+import {
+  InjectedQueryWrapperProps,
+  QueryWrapper,
+} from "../../commonComponents/QueryWrapper";
+import "./PendingOrganizationsList.scss";
+import Alert from "../../commonComponents/Alert";
+import { showNotification } from "../../utils";
+import Checkboxes from "../../commonComponents/Checkboxes";
+import Button from "../../commonComponents/Button";
+
+export const ORGANIZATIONS_QUERY = gql`
+  query GetUnverifiedOrganizations($identityVerified: Boolean) {
+    organizations(identityVerified: $identityVerified) {
+      id
+      name
+      externalId
+      identityVerified
+    }
+  }
+`;
+export const SET_ORG_IDENTITY_VERIFIED_MUTATION = gql`
+  mutation SetOrgIdentityVerified($externalId: String!, $verified: Boolean!) {
+    setOrganizationIdentityVerified(
+      externalId: $externalId
+      verified: $verified
+    )
+  }
+`;
+
+interface Props {
+  data: any;
+  refetch: () => void;
+}
+
+export const DetachedPendingOrganizationsList: any = ({
+  data,
+  refetch,
+}: Props) => {
+  const orgs = data?.organizations || [];
+
+  const [verifiedOrgExternalIds, setVerifiedOrgExternalIds] = useState(
+    new Set()
+  );
+
+  function orgRows(orgs: any) {
+    if (orgs.length === 0) {
+      return (
+        <tr>
+          <td>No results</td>
+        </tr>
+      );
+    }
+
+    return [...orgs].map((o) => {
+      return (
+        <tr key={o.id} className={classnames("sr-org-row")}>
+          <th scope="row">{o.name}</th>
+          <th scope="row">{o.externalId}</th>
+          <td>
+            <Checkboxes
+              onChange={(e) => {
```

Comment:
This is sorta nitpicky, but it might be a little cleaner to define this change handler outside of the markup